### PR TITLE
Fix new warnings in Rust nightly-2019-01-11

### PR DESCRIPTION
**Pull request by SimonSapin**
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/22674)

### CODE REVIEW

File: components/style_traits/values.rs [hunk 1]
```diff
@@ -450,7 +432,7 @@ impl_to_css_for_predefined_type!(::cssparser::RGBA);
 impl_to_css_for_predefined_type!(::cssparser::Color);
 impl_to_css_for_predefined_type!(::cssparser::UnicodeRange);
 
-#[macro_export]
+/// Define an enum type with unit variants that each corrsepond to a CSS keyword.
```
Comment (lqd) [inline components/style_traits/values.rs]: tiny typo on "corrsepond"

```suggestion
/// Define an enum type with unit variants that each correspond to a CSS keyword.
```

Comment (SimonSapin) [reply components/style_traits/values.rs]: Applied, thanks!

**Review by emilio — APPROVED** [review]

**Pull request merged**